W-22960628 Add AI Features and Data Usage page (#456)
mulesoft/docs-general: modules/ROOT/pages/ai-features-and-data-usage.adoc

= AI Features and Data Usage

> AI in MuleSoft Anypoint Platform is built on data. Before you use a feature with AI in MuleSoft, it is important to understand what data is used and how.

Reviewer: Suggestion for intro:
MuleSoft AI features use your data to generate responses. Understanding what data each feature uses helps you make informed decisions about privacy, compliance, and how you work with AI in MuleSoft.

> All MuleSoft AI features access large language models (LLMs) within the Salesforce Trust Boundary. Data that you submit to MuleSoft AI features is processed securely within Salesforce-managed boundaries and routed through your connected Salesforce organization for tracking and billing.

Reviewer: Suggestion:
All MuleSoft AI features access LLMs within the Salesforce Trust Boundary, a set of security and data governance controls that keeps your data within Salesforce-managed infrastructure. MuleSoft AI features process data you submit securely within Salesforce-managed boundaries and route it through your connected Salesforce organization for tracking and billing.

> The following table lists the Customer Data you submit to our services, as defined in our https://www.salesforce.com/en-us/wp-content/uploads/sites/4/documents/legal/Salesforce_MSA.pdf[Main Services Agreement (MSA)^], and the usage data relating to your interactions with MuleSoft AI features. Customer Data that you submit to MuleSoft AI features is not used to train AI models or shared across customers.

Reviewer: ```suggestion
This table lists the Customer Data you submit to our services, as defined in our https://www.salesforce.com/en-us/wp-content/uploads/sites/4/documents/legal/Salesforce_MSA.pdf[Main Services Agreement (MSA)^], and usage data about your interactions with MuleSoft AI features. Customer Data that you submit to MuleSoft AI features is not used to train AI models or shared across customers.
```

For more information about data usage, see Customer Data and Usage Data in the MSA. For billing information, see https://help.salesforce.com/s/articleView?id=ai.generative_ai_usage.htm&type=5[Agentforce and Generative AI Usage and Billing^].

[cols="1,1,3a,2a",options="header"]
|===
| Product | Feature | Data Used | Generative AI Model Used

| Intelligent Document Processing (IDP)
| Document Analysis with Custom Schemas
a|

> This feature uses document images that you upload in base64-encoded format and the extracted text from them, along with natural language prompts and questions that you provide. It also uses document schema metadata that you define, such as field names, descriptions, and data types, and sample data that conveys the expected output format.

Reviewer: ```suggestion
This feature uses document images that you upload and the extracted text from them, along with natural language prompts and questions that you provide. It also uses document schema metadata that you define, such as field names, descriptions, and data types, and sample data that conveys the expected output format.
```

| Anthropic Claude Sonnet 3.5 with OpenAI GPT-4o Mini for planning (via Amazon Bedrock)

| DataWeave Generation
a|
This feature uses input and output sample data that you provide, such as JSON, XML, and CSV, along with optional structure metadata and natural language instructions. It grounds responses with MuleSoft DataWeave function and operator documentation.
| OpenAI GPT-4o

| Anypoint Exchange
| API Documentation Generation
a|
This feature uses API specifications that you upload, such as RAML, OAS, SOAP, WSDL, AsyncAPI, GraphQL, and gRPC, along with asset metadata such as name, description, tags, and version.
| OpenAI GPT-4o Mini

| Runtime Manager
| Diagnostics Agent
a|
This feature uses Java thread dumps from your deployed applications and the Diagnostic Information Analysis File (DIAF), which includes application metadata, performance statistics, error messages and stack traces, and recent log entries. It also uses CloudHub or CloudHub 2.0 instance information and deployment configuration.
| Google Gemini 2.5 Flash

> |===

Reviewer: Can we add a See Also?
== See also

* xref:usage-reports.adoc[]
* xref:pricing.adoc[]
* xref:usage-metrics.adoc[]
